question/tests/behat/columnsortorder.feature

```diff
@@ -0,0 +1,28 @@
+@core @core_question @javascript
+Feature: An plugin column can be reordered and displayed in the question bank view.
+  In order to reorganise the question bank view columns
+  As an admin
+  I need to be able to reorder them
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
+      | teacher1 | Teacher | 1 | teacher1@example.com |
+    And the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1 | weeks |
+    And the following "question categories" exist:
+      | contextlevel | reference | name           |
+      | Course       | C1        | Test questions |
+    And the following "questions" exist:
+      | questioncategory | qtype | name                        | user  | questiontext                  | idnumber  |
+      | Test questions   | essay | Test question to be seen | admin | Write about whatever you want | numidnum</1 |
+    And I log in as "admin"
+
+  @javascript
+  Scenario: Reordering question bank columns
+    When I navigate to "Plugins > Question bank plugins > Column sort order" in site administration
+    And I drag ".item" "css_element" and I drop it in ".list" "css_element"
+    And I am on "Course 1" course homepage
+    And I navigate to "Question bank > Questions" in current page administration
+    Then ".creatorname" "css_element" should appear before ".modifiername" "css_element"
```

Comment:
can you please check if this element test checks whether the css element immediately before the specified element or it also accepts if there is another element in the middle of those two elements, because logically its appearing before the other one but functionally what will not be correct.
P.S. I didn't do a review in this PR, I was curious and wanted to comment on this)

Reply:
You are right, it accepts columns in between, I specified where to drop in the code what should solve the problem, thank you !